src/olympia/editors/tests/test_models.py

```diff
@@ -56,24 +64,27 @@ def create_addon_file(name, version_str, addon_status, file_status,
     file_defaults.update(file_kw)
     file_ = File.objects.create(**file_defaults)
     if created:
-        vr.update(created=created)
+        version.update(created=created)
         file_.update(created=created)
     # Update status *after* we are done creating/modifying version and files:
-    Addon.with_unlisted.get(pk=ad.id).update(status=addon_status)
-    return {'addon': ad, 'version': vr, 'file': file_}
+    addon = Addon.with_unlisted.get(pk=addon.id)
+    addon.update(status=addon_status)
+    return {'addon': addon, 'version': version, 'file': file_}
 
 
 def create_search_ext(name, version_str, addon_status, file_status,
                       listed=True):
-    ad, created_ = Addon.with_unlisted.get_or_create(
+    addon, created_ = Addon.with_unlisted.get_or_create(
         name__localized_string=name,
         defaults={'type': amo.ADDON_SEARCH, 'name': name, 'is_listed': listed})
-    vr, created_ = Version.objects.get_or_create(addon=ad, version=version_str)
-    File.objects.create(version=vr, filename=u"%s.xpi" % name,
+    version, created_ = Version.objects.get_or_create(
+        addon=addon, version=version_str)
```

Comment:
Doubt it matters at this point, but not setting the channel here might bite us later.